data/data/agent/systemd/units/agent-auth-token-status.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Service that checks if agent auth token is unexpired and displays a message if expired.
+Wants=network-online.target
+After=network-online.target agent-interactive-console.service
+ConditionPathExists=/etc/assisted/add-nodes.env
```

Comment:
Makes sure the service runs only for the `addnodes` workflow

Reply:
WIP: I think even if the tokens do not expire for day1, it won't hurt if we run this service on the nodes boot up on day1 as well in that case the token won't expire and will not error for token expiry.

Reply:
That way it will be easy to manage the various systemd services and won't need a lot of code changes.

Reply:
This service won't run on the nodes when its day1